CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java

```diff
@@ -342,23 +372,36 @@ private static int processVertices(ImportDefinition definition, GraphWriteMethod
 
             final String[] row = data.get(i);
             if (filter == null || filter.passesFilter(i - 1, row)) {
-                // Count the number of processed rows to notify in the status message
-                ++importedRows;
-                final int vertexId = graph.addVertex();
 
-                for (final ImportAttributeDefinition attributeDefinition : attributeDefinitions) {
-                    attributeDefinition.setValue(graph, vertexId, row, (i - 1));
-                }
+                try {
+                    final int vertexId = graph.addVertex();
+
+                    for (final ImportAttributeDefinition attributeDefinition : attributeDefinitions) {
+                        attributeDefinition.setValue(graph, vertexId, row, (i - 1));
+                    }
 
-                if (initialiseWithSchema && graph.getSchema() != null) {
-                    graph.getSchema().completeVertex(graph, vertexId);
+                    if (initialiseWithSchema && graph.getSchema() != null) {
+                        graph.getSchema().completeVertex(graph, vertexId);
+                    }
+
+                    // Count the number of processed rows to notify in the status message
+                    ++importedRows;
+
+                } catch (Exception ex) {
```

Comment:
are you able to change this catch to catch the specific exceptions it is catching (also can be final)